cmake/Modules/FindComputeCpp.cmake

```diff
@@ -50,8 +50,8 @@ elseif ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang")
       message(STATUS "host compiler - clang ${CMAKE_CXX_COMPILER_VERSION}")
     endif()
 else()
-  message(WARNING
-    "host compiler - Not found! (ComputeCpp supports GCC and Clang, see readme)")
+  set (COMPUTECPP_PLATFORM_SPECIFIC_ARGS "-fno-ms-compatibility")
+  #message(WARNING "host compiler - Not found! (ComputeCpp supports GCC and Clang, see readme)")
```

Comment:
This can be removed. Arguably this whole if-else chain could be removed, as other compilers are unlikely to work (though maybe icc would? And if it is capable, why would we stop it?)